src/app/test/test-reports.service.ts

```diff
@@ -38,7 +40,7 @@ export class TestReportsService {
       });
   }
 
-  async matchRerunResults(reports: TestListItem[]) {
+  async matchRerunResults(reports: TestListItem[]): Promise<TestListItem[]> {
     const oldReports: TestListItem[] = await firstValueFrom(this.testReportsSubject);
     const filteredReports: TestListItem[] = oldReports.filter((r: TestListItem) => !!r.reranReport);
```

Comment:
```suggestion
    const filteredReports: TestListItem[] = oldReports.filter((report: TestListItem) => !!report.reranReport);
```

Reply:
Ik heb het in VSCode nagedaan - dan weet ik zeker dat het hernoemen consequent gedaan is.